scripts/example_stac_calibration_specific_items.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+import argparse
+
+from paint import PAINT_ROOT
+from paint.data.stac_client import StacClient
+from paint.util import set_logger_config
+
+set_logger_config()
+
+if __name__ == "__main__":
+    # Read in arguments.
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--output_dir",
+        type=str,
+        help="Path to save the downloaded data.",
+        default=f"{PAINT_ROOT}/download_calibration_test",
+    )
```

Comment:
`pathlib.Path` objects (also see above, not sure whether this would mess with some functions expecting strings).